bin/input_assure.py

```diff
@@ -19,38 +19,45 @@ def check_inputs(json_file, sample_id, address, output_error_file, output_json_f
     with open_file(json_file, "rt") as f:
         json_data = json.load(f)
 
+    print(json.dumps(json_data, indent=4))
+
+    # Extract the profile from the json_data
+    profile = json_data.get("data", {}).get("profile", {})
+    # Check for multiple keys in the JSON file and define error message
+    keys = list(profile.keys())
```

Comment:
This is not something that needs to change, just something I was thinking through and am just writing out my thoughts here as a reminder/in case there's any issues you could see.

I believe you are adding the keys (sample identifiers) to a list in order to select the first one from the list below, correct (in the situation with multiple keys)? A dictionary technically has no first element and according to https://docs.python.org/3/tutorial/datastructures.html#dictionaries keys are added to a list in insertion order:

>Performing list(d) on a dictionary returns a list of all the keys used in the dictionary, in insertion order (if you want it sorted, just use sorted(d) instead).

So I don't believe e.g., that the first sample identifier will necessarily be e.g., the first identifier listed in the file. But, in theory should be repeatable (assuming insertion order is repeatable), which I think is all we really need unless you can see any other issues that could come up? You think we should just sort the list (`sorted()`) to guarantee a repeatable order?

Reply:
I've considered this and agree that switching to `sorted()` is the right choice. I'm thinking ahead about potential future changes, like with `sample_name`, and believe it's important to ensure a guaranteed and repeatable order to avoid future conundrums 😄 
Changed here: [6f7f59d](https://github.com/phac-nml/gasclustering/pull/28/commits/6f7f59d09714667e004dc55a70f6f7052ec5c7bd)

Reply:
Thanks so much for considering this suggestion 😄